sickbeard/show_name_helpers.py

```diff
@@ -59,24 +78,21 @@ def filterBadReleases(name, parse=True):
     #    return False
 
     # if any of the bad strings are in the name then say no
+    ignore_words = list(resultFilters)
     if sickbeard.IGNORE_WORDS:
-        resultFilters.extend(sickbeard.IGNORE_WORDS.split(','))
```

Comment:
I think I've fixed a memory leak here, because content of global variable `resultFilters` is growing at each run of `filterBadReleases`. Fix is to create a new list from `resultFilters`.